### ISSUE TITLE
Empty math parameter in TexImage crashes

### ISSUE THREAD

**Issue by carloshgsilva** [amake/flutter_tex_js#2]
Problem:
If the parameter math get an empty string it crashes and make the others TeXImage don't load.

Code:
```dart
TexImage('');
```

Console: 
```
D/AMK     (30054): onReady called; thread=Thread[JavaBridge,7,main]
D/AMK     (30054): Executing JavaScript: setNoWrap(false); setWidth('333.42857142857144px'); setColor('rgba(0,0,0,0.8666666666666667)'); setFontSize('24.0px'); render('', true);
D/AMK     (30054): Taking snapshot of [0.0, 0.0, 0.0, 0.0], scaled to [0, 0, 0, 0]
W/System.err(30054): java.lang.IllegalArgumentException: width and height must be > 0
W/System.err(30054): 	at android.graphics.Bitmap.createBitmap(Bitmap.java:1033)
W/System.err(30054): 	at android.graphics.Bitmap.createBitmap(Bitmap.java:1000)
W/System.err(30054): 	at android.graphics.Bitmap.createBitmap(Bitmap.java:950)
W/System.err(30054): 	at android.graphics.Bitmap.createBitmap(Bitmap.java:911)
W/System.err(30054): 	at com.madlonkay.flutter_tex_js.TexRenderer.takeSnapshot(TexRenderer.kt:149)
W/System.err(30054): 	at android.os.MessageQueue.nativePollOnce(Native Method)
W/System.err(30054): 	at android.os.MessageQueue.next(MessageQueue.java:326)
W/System.err(30054): 	at android.os.Looper.loop(Looper.java:160)
W/System.err(30054): 	at android.os.HandlerThread.run(HandlerThread.java:65)
I/chromium(30054): [INFO:CONSOLE(54)] "Uncaught Error: Java exception was raised during method invocation", source: file:///android_asset/ (54)
```

**Comment by amake** [comment]
Thanks. I've addressed this by rendering a `Text('')` if the input is empty. I chose this because I imagine it's desirable for the `TexImage` to take up approximately one text line of height even if it's empty, for consistency with surrounding elements.

If you have something else in mind, please let me know.

**Comment by amake** [comment]
I just released 0.1.5 with the fix. Please let know if there are any further issues.
